SECURITY.md

```diff
@@ -0,0 +1,19 @@
+## **Reporting a Vulnerability**
+
+We take the security of our projects seriously. If you discover a security vulnerability,
+we appreciate your help in disclosing it to us in a responsible manner.
+
+**Please DO NOT open a public JIRA/GitHub issue.**
+
+Instead, please report the vulnerability directly as described at
+the [Security Contacts and Procedures](https://access.redhat.com/security/team/contact/#contact).
+
+### **Supported Versions**
+
+We only provide security updates for the **latest stable release** and releases that are in the limited-support state.
```

Comment:
Is this extracted from somewhere else? Because in my mind "limited-support" meant that we _might_ provide security updates, but don't provide guarantees. It's entirely possible the update would be provided for "supported builds" only (SP, ...).

Reply:
In particular:

> Backports to these series are highly restricted and releases are not guaranteed.

> Bug-fixes and improvements are only applied to latest-stable/stable and development (and sometimes to limited-support) series.

> Only development and latest-stable/stable series are actively released. Occasional releases of limited-support series may happen, but should not be expected.